indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -23,6 +23,47 @@ $(document).ready(function() {
         });
     });
 
+    var threeRowsHeight = 69;
```

Comment:
Or I could use a global JS variable to initialize the height and then check if the variable exists before initializing it.